src/index.ts

```diff
@@ -473,12 +471,11 @@ export function slashTimelockUnbondedTransaction(
  * - psbt: The partially signed transaction (PSBT).
  *
  * @param {Object} scripts - The scripts used in the transaction. e.g slashingScript, unbondingTimelockScript
- * @param {Transaction} transaction - The original staking transaction.
+ * @param {Transaction} stakingTransaction - The original staking transaction.
```

Comment:
```suggestion
 * @param {Transaction} unbondingTransaction - The unbonding transaction.
```

This should not be the original staking transaction. It should be the unbonding transaction. The naming in the function parameter is a mistake.

Reply:
Done